shoplist/src/main/java/com/netguru/codereview/network/ShopListRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network
+
```

Comment:
it's better if we make the interface "ShopListRepository"  and provide the implementation for this interface